Web_Development/Javascript/Topics/3.Web_Browser/README.md

```diff
@@ -1 +1,82 @@
-# To Do
\ No newline at end of file
+# Web Browser In Javascript
+
+**Browser** is the fundamental host environment for the JavaScript. It is a platform that extends the core JavaScript and provides additional features through specific objects and functions.
+</br>
+* The core object for **JavaScript** in web browser is the window object. It represents web browser window and simultaneously the ECMAScript Global object. 
```

Comment:
```suggestion
* The global object for **JavaScript** in web browser is the window object. It represents web browser window and simultaneously the ECMAScript Global object. 
```

Write in a way that resembles with javascript lingos, so that if one reads this doc will not get confused by reading additional words